### ISSUE TITLE
Maintainer

### ISSUE THREAD

**Issue by gabrielbull** [johanneslumpe/react-native-selectablesectionlistview#20]
Please add a maintainer to this project that will fix issues and merge pull requests. Thanks!

**Comment by johanneslumpe** [comment]
@gabrielbull Do you want to be added as a collaborator?

**Comment by gabrielbull** [comment]
Yes please. I don't have much time on my hand but I'll be at least able to make this library work with RN@latest.

Cheers 🍻!
